### PR TITLE
Usability improvments

### PR DESCRIPTION
**Pull request by ansigroup**
- fix missing data for account history
- improve for asset history (assetids as links, accounts as links)
- improve for account history  (assetids as links, accounts as links, better representation for idata/mdata/ixdata/mxdata/exdata, seriesObjectAssetId as link)

### CODE REVIEW

File: hyperion-explorer/src/app/search-results/account/account.component.html [hunk 1]
```diff
@@ -346,7 +346,7 @@ <h3 style="margin-top: 0;">
 
             <!-- DETAILED ACTION DATA VIEW-->
             <ng-container *ngIf="detailedView">
-
+			
               <div *ngFor="let item of action['act']['data'] | keyvalue"
```
Comment (AlexGriffith-Voice) [inline hyperion-explorer/src/app/search-results/account/account.component.html:350]: remove extra spacing

File: hyperion-explorer/src/app/utils/sa2json.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+				r[k] = '' + o[k];
+			}
+		});
+		return r;
+    }
+
+}
+//export SA2JSON;
+//exports.default = SA2JSON;
+module.exports = {SA2JSON};
```
Comment (AlexGriffith-Voice) [inline hyperion-explorer/src/app/utils/sa2json.js:142]: new line

**Review by AlexGriffith-Voice — APPROVED** [review]
I think there should be cleanup on the random extra spaces and new lines but otherwise go for it. Also don't forget to build out the files before merging.

File: hyperion-explorer/src/app/search-results/simple-assets/simple-assets.component.ts [hunk 3]
```diff
@@ -65,5 +65,23 @@ export class SimpleAssetsComponent implements OnInit {
 
   stringifyObject(subitem: object): string {
     return JSON.stringify(subitem, null, 2);
+  }
+
+  isJSON(item) {
```
[the remark this answers is not in the document]
Comment (ansigroup) [reply hyperion-explorer/src/app/search-results/simple-assets/simple-assets.component.ts]: fixed, thanks

File: hyperion-explorer/src/app/utils/sa2json.js (hunk 2 above)
Comment (ansigroup) [reply hyperion-explorer/src/app/utils/sa2json.js]: fixed, thanks

**Pull request merged**